test-d/foundry/foundry.js/clientDocuments/chatMessage.test-d.ts

```diff
@@ -28,12 +28,12 @@ if (game instanceof Game) {
     ChatMessage.getSpeaker({
       scene: game.scenes?.active,
       actor: game.user?.character,
-      token: new Token(new TokenDocument()),
+      token: new TokenDocument(),
       alias: 'Mario'
     })
   );
 }
-expectType<ChatSpeakerData['_source']>(ChatMessage.getSpeaker({ token: new Token(new TokenDocument()) }));
+expectType<ChatSpeakerData['_source']>(ChatMessage.getSpeaker({ token: new TokenDocument() }));
```

Comment:
What's the reason for these changes?

Reply:
Earlier, it was only possible to pass a `Token`. 0.8.9 changed it so that you can actually pass both a `Token` or a `TokenDocument` but only the variant for `TokenDocument` is documented. passing a `Token` will trigger a warning. While it technically still works, it’s not something you are supposed to do anymore, so I just typed it as `TokenDocument`. If you still think that we should allow both, we can do that.

Reply:
No this is fine then to me.